src/qtiItem/helper/util.js

```diff
@@ -122,10 +122,10 @@ var util = {
         }
 
         if (_.isArray(collectionNames)) {
-            _.each(collectionNames, function(collectionName) {
+            _.each(collectionNames, function (collectionName) {
                 //get collection to search in (resolving case like interaction.choices.0
                 var collection = element;
-                _.each(collectionName.split('.'), function(nameToken) {
+                _.each(collectionName.split('.'), function (nameToken) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
                _.forEach(collectionName.split('.'), function (nameToken) {
```